concentration/ViewControllers/MenuViewController.swift

```diff
@@ -8,16 +8,16 @@
 import SpriteKit
 
 struct IASceneType {
-    let scene: SKScene
+    let scene: SKScene.Type
```

Comment:
this is not right place to change it, we will create new task and branch for his bug